client/src/components/ConnectStripe/ConnectStripe.tsx

```diff
@@ -0,0 +1,47 @@
+import Button from '@mui/material/Button';
+import { useAuth } from '../../context/useAuthContext';
+import SettingHeader from '../../components/SettingsHeader/SettingsHeader';
+import { Box } from '@mui/material';
+import getLink from './../../helpers/APICalls/stripe';
+import { StripeLink } from '../../interface/StripeLink';
+
+interface StripeProps {
+  header: string;
+}
+
+export const ConnectStripe: React.FC<StripeProps> = ({ header }) => {
+  const { profile } = useAuth();
+
+  const clickHandler = async () => {
+    await getLink().then((data: StripeLink) => {
```

Comment:
If it fails could we send a message too? Just something like "An error occured while connecting Stripe". Take a look at the useSnackbar context which we can make use of here.

Reply:
updated in latest version - thanks!